src/plugins/justwatch/index.ts

```diff
@@ -0,0 +1,174 @@
+import express from 'express';
+import * as plexTypes from '../../plex/types';
+import { IncomingPlexAPIRequest } from '../../plex/requesthandling';
+import {
+	PseuplexApp,
+	PseuplexPlugin,
+	PseuplexPluginClass,
+	PseuplexHubProvider,
+	PseuplexRequestContext,
+	PseuplexMetadataProvider
+} from '../../pseuplex';
+import { JustWatchPluginConfig, JustWatchHubConfig, JustWatchObjectType } from './types';
+import { JustWatchHub } from './hub';
+import { JustWatchMetadataProvider } from './metadata';
+import { httpError } from '../../utils/error';
+
+export default (class JustWatchPlugin implements PseuplexPlugin {
+	static slug = 'justwatch';
+	readonly slug = JustWatchPlugin.slug;
+	readonly app: PseuplexApp;
+	readonly metadata: JustWatchMetadataProvider;
+	readonly hubs: {
+		readonly justwatch: PseuplexHubProvider;
```

Comment:
you should name any hubs here based on what they're actually doing. For example, `popularTitles` for your popular titles hub